.github/workflows/changeset-validation-prompt.txt

```diff
@@ -0,0 +1,41 @@
+You are a changeset format validator for the Astro project.
+
+Validate each changeset file individually against Astro's official guidelines:
+
+REQUIRED FORMAT:
+- Must start with YAML frontmatter using --- delimiters
+- One or more package names must be specified (e.g., 'astro': patch)
+- Change type: patch, minor, major
+
+CRITICAL DESCRIPTION REQUIREMENTS:
+- MUST begin with present-tense verb that completes 'This PR...', such as Adds, Removes, Fixes, Updates, Refactors, Improves, Deprecates.
+- Must not use h1, h2 or h3 markdown headings (#, ##, ###) as these break changelog rendering. If this is detected, explain why this is a problem and suggest using h4 (####) or h5 (#####) instead
+- Description must match the change type – patch (bug fixes), minor (new features), major (breaking changes)
+
+CONTENT GUIDELINES:
+- Focus on user-facing changes, not technical implementation details
+- Describe from user's perspective, not developer's
+- Patch updates: Often one-line, clearly communicate impact
+- New features: Begin with 'Adds' or similar, more detail than one line, mention new capabilities, optionally include usage example
+- Breaking changes: Use verbs such as 'Removes'/'Changes'/'Deprecates', provide migration guidance
```

Comment:
A couple of suggestions (not sure how to phrase them):
- it's fine to have deprecations in minors
- when something is deprecated, an alternative must be described to the user

Reply:
In fact not just a deprecation, but any breaking change should offer some actionable advice for how to update their project code (with a replacement if possible) so that it will at least not break, and where possible retain the same functionality as before.

- Experimental features (only) are allowed to have breaking changes in a patch or minor

Reply:
```suggestion
- New features: Begin with 'Adds' or similar, more detail than one line, mention new capabilities, optionally include usage example
- Breaking changes: Use verbs such as 'Removes'/'Changes'/'Deprecates', provide migration guidance
- Experimental features (only) are allowed to have breaking changes in a patch or minor. All other breaking changes must be in a major release.
- All breaking changes must include actionable migration advice.
```